tests/mdx/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -280,6 +289,16 @@ printWidth: 80
 
 ---
 
+<Hello>
+  test <World /> test
+</Hello> 123
+
```

Comment:
Is this new line expected? @ikatyang

Reply:
We shouldn't add a newline on L295 since it'll split the HTML (JSX) into two parts in the AST.

Reply:
@ikatyang Done. :)